things/api.py

```diff
@@ -476,30 +485,46 @@ def today(**kwargs):
     """
     Read Today's tasks into dicts.
 
-    Note: This might not produce desired results if the Things app hasn't
-    been opened yet today and the yellow "OK" button clicked for new tasks.
-    In general, you can assume that whatever state the Things app was in
-    when you last opened it, that's the state reflected here by the API.
+    Note: This method is a bit tricky. The API reflects the status of the database
+    when you last opened it with the app. However, there is the concept of "yellow"
+    time-related tasks. Some cases (e.g., scheduled or due tasks) are handled by the
+    API and are automatically shown. In other cases, however, such as with repeating
+    tasks, the database gets modified when opened with the app. These cases are not
+    covered before the app got launched.
 
     See `things.api.tasks` for details on the optional parameters.
     """
-    database = pop_database(kwargs)
-    if not database.was_modified_today():  # pragma: no cover
-        print(
-            "[NOTE] The results reflect the state of the Things app "
-            "when it was last run. If the results seem out of date, "
-            "then run the Things app and click the yellow 'OK' button "
-            "to update Today's to-dos and projects.",
-            file=sys.stderr,
-        )
-    return tasks(
+    regular_today_tasks = tasks(
         start_date=True,
         start="Anytime",
         index="todayIndex",
-        database=database,
         **kwargs,
     )
 
+    unconfirmed_scheduled_tasks = tasks(
+        start_date="past",
+        start="Someday",
+        index="todayIndex",
+        **kwargs,
+    )
+
+    unconfirmed_overdue_tasks = tasks(
+        start_date=False,
+        deadline="past",
+        start="Anytime",
```

Comment:
Wouldn't `Inbox` and `Someday` be possible too? Maybe the start is irrelevant here.

Reply:
ping @AlexanderWillner